modules/enet/doc_classes/NetworkedMultiplayerENet.xml

```diff
@@ -124,6 +124,23 @@
 				Configure the [CryptoKey] to use when [member use_dtls] is [code]true[/code]. Remember to also call [method set_dtls_certificate] to setup your [X509Certificate].
 			</description>
 		</method>
+		<method name="set_peer_timeout">
+			<return type="void">
+			</return>
+			<argument index="0" name="id" type="int">
+			</argument>
+			<argument index="1" name="timeout_limit" type="int">
+			</argument>
+			<argument index="2" name="timeout_min" type="int">
+			</argument>
+			<argument index="3" name="timeout_max" type="int">
+			</argument>
+			<description>
+				Sets the timeout parameters for a peer.	The timeout parameters control how and when a peer will timeout from a failure to acknowledge reliable traffic. Timeout values are expressed in milliseconds.
+
```

Comment:
Should not have empty lines (== empty paragraph).

Reply:
always forget this is not markdown :smirk_cat: